tests/core/pyspec/eth2spec/test/helpers/bls_to_execution_changes.py

```diff
@@ -17,10 +17,10 @@ def get_signed_address_change(spec, state, validator_index=None, withdrawal_pubk
     if to_execution_address is None:
         to_execution_address = b'\x42' * 20
 
-    if fork_version is None:
-        domain = spec.get_domain(state, spec.DOMAIN_BLS_TO_EXECUTION_CHANGE)
-    else:
-        domain = spec.compute_domain(spec.DOMAIN_BLS_TO_EXECUTION_CHANGE, fork_version, state.genesis_validators_root)
+    domain = spec.compute_domain(
```

Comment:
I would suggest that although the default (passing in None) for `genesis_validators_root` should be the one read from state, I also suggest we can override and do an explicit test with a mismatch

Reply:
done here https://github.com/ethereum/consensus-specs/pull/3206/commits/d4eaf4bff4eac9d7a84455f1939b0be87c2e35e3